### PR TITLE
fix: change endpoint in dev env

### PR DESCRIPTION
**Pull request by yjkimjunior**
change frontend w.r.t. https://github.com/paritytech/Nomidot/pull/340

### CODE REVIEW

File: front/gatsby/src/apollo.ts [hunk 1]
```diff
@@ -13,15 +13,15 @@ const httpLink = new HttpLink({
   uri: `${
     process.env.NODE_ENV === 'production'
       ? 'http://nomidot-server.nomidot-staging:4000'
```
Comment (Tbaut) [inline front/gatsby/src/apollo.ts:15]: I don't think this will work in prod. This is a local url that only works in staging.

Comment (niklabh) [reply front/gatsby/src/apollo.ts:15]: This works in k8s. Internal urls are of format service-name.namespace

Comment (Tbaut) [reply front/gatsby/src/apollo.ts:15]: But `prod` and `staging` clusters are not connected, in the prod cluster, you won't be able to access `http://...**nomidot-staging**` that lives in the other cluster.

Comment (yjkimjunior) [reply front/gatsby/src/apollo.ts:15]: ah indeed, so I'd change `process.env.NODE_ENV === 'staging'` to use `'http://nomidot-server.nomidot-staging:4000'`, then use another one for `production` once we get there.

Is that how you do it in Polkassembly @Tbaut?

Comment (Tbaut) [reply front/gatsby/src/apollo.ts:15]: What we have in Polkassembly is `https://test.nodewatcher-server.polkassembly.io/` for prod and the local `http://nomidot-server.nomidot-staging:4000` for test. Although the `test.nodewatcher-server.polkassembly.io` is available from anywhere, so you could use that both in prod and test, we've tested it successfully yesterday.

**Pull request merged**